File: modulo1/segmento3/apunte6/Ejercicio3.web/ejercicio3.js
function ejercicio3() {
	

		alert("Area de un rectangulo");
        base = parseInt(prompt('Ingrese base del rectangulo: '));
		altura = parseInt(prompt('Ingrese altura del triangulo: '));
		if (base > 0 && altura > 0) {
        let area = base * altura;
		alert ('El area del rectangulo es :' + area);
		
		}
		else {
			alert ('Error! Los numeros ingresados deben ser mayores a  0 ')
			let otro = false;

		while(otro == false) {
		alert("Area de un rectangulo");
        base = parseInt(prompt('Ingrese base del rectangulo: '));
		altura = parseInt(prompt('Ingrese altura del triangulo: '));
		
		}
	}
}

File: modulo1/segmento3/apunte6/Ejercicio3.java
import java.util.Scanner;

public class Ejercicio3 {
	public static void main(String args[]){
		Scanner keyboard = new Scanner(System.in);

			Double result , num1, num2;
				do{
					

					System.out.print("Ingrese base del rectangulo: ");
					num1 = keyboard.nextDouble();

					System.out.print("Ingrese altura del rectangulo : ");
					num2 = keyboard.nextDouble();
					if(num1 > 0 && num2 > 0) {
					result = num1 * num2;

					System.out.println(" ");
						
					System.out.println("El area del rectangulo es : " + result);
					} else  {
						System.out.println("Error! Area y altura deben ser mayores a cero ! Vuelve a ingresar ");
					}
					System.out.println("==================================");
			} while(num1 <= 0 || num2 <= 0);
	}
}